api/v1alpha1/openstackclusterstackrelease_types.go

```diff
@@ -28,8 +28,25 @@ type OpenStackClusterStackReleaseSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of OpenStackClusterStackRelease. Edit openstackclusterstackrelease_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// The name of the cloud to use from the clouds secret
+	CloudName string `json:"cloudName"`
+
+	// IdentityRef is a reference to a identity to be used when reconciling this cluster
+	IdentityRef OpenStackIdentityReference `json:"identityRef,omitempty"`
```

Comment:
In the CAPO, there is *OpenStackIdentityReference, whats the difference?

Reply:
If I am correct `*` in go is a pointer to the specified type. I think the primary difference is whether `IdentityRef` is a pointer or a direct instance of `OpenStackIdentityReference`. Using a pointer allows the possibility of a null value, while using an actual instance does not

Reply:
Thank you, and why you don't use pointer?

Reply:
I think, it is up to us. If we want to have possibility of a `null` value for this field or not.

Reply:
I think that it can also speed up code when we work with references so the object doesn't need to be copied